cmd/batch-handlers.go

```diff
@@ -57,6 +58,11 @@ import (
 
 var globalBatchConfig batch.Config
 
+const (
+	// Keep the completed/falied job stats 3 days before removing it
+	oldJobsExpiration = 3 * 24 * time.Hour
```

Comment:
3 days? 24hr is enough I think.

Reply:
@harshavardhana the batch job is a long running job, I doubt users will check the batch status everyday, then once the job batch completes or fails + 24 hours, there is no trace at all about what happened. Maybe we should not remove in the first place and add a new API to remove batch reports

Reply:
Yeah for that they must configure notifications, we shouldn't have to carry this around.

Can you check what AWS S3 does?

Reply:
did you verify? @vadmeste

Reply:
@harshavardhana as far as I found, users configure the way to store the result of the job batch, like saving the report in an s3 bucket, but there is no automatic cleaning here

Reply:
yeah we can provide such an option as well @vadmeste keeping this around for 3 days for now is okay.